### ISSUE TITLE
Bot quits when catch daily limit is met

### ISSUE THREAD

**Issue by pogarek** [PokemonGoF/PokemonGo-Bot#5795]
When limit is reached, pokemon bot quits. Shouldn't it just continue work to spin pokestops, walk eggs , buddy etc ?

The file cell_workers/pokemon_catch_worker.py  has the following currently:

                self.emit_event('catch_limit', formatted='WARNING! You have reached your daily catch limit')
                sys.exit(2)
                break

**Comment by Gobberwart** [comment]
Someone was working on changing this before the api change, so you could choose whether to exit the bot, or keep farming without catching anything. I do not know what happened to that. Possibly abandoned by the person who was working on it? For now, it's working as intended/designed, so not a bug. Added "Feature Request" tag.

**Comment by reigelgallarde** [comment]
if you wanted it to continue to work, why put a limit?

**Comment by reigelgallarde** [comment]
ahh fast reply without thinking... sorry... I now get it...

this would be great addition...

**Comment by reigelgallarde** [comment]
commenting out `# sys.exit(2)` helps...
here's the output....


![Uploading Capture8.PNG…](http://i.imgur.com/1guOABC.png)

**Comment by pogarek** [comment]
Yes, I have commented it out for me. But I don't get the idea to kill the bot so quickly when limit is reached :)
